test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -93,6 +104,44 @@ export async function getSessionScopes(
   return JSON.parse(await getSessionRawResult.getText());
 }
 
+/**
+ * Use dapp UI to send custom addresses to request scope for.
```

Comment:
```suggestion
 * Use dapp UI to add account addresses to `wallet_createSession` request.
```

Reply:
resolved in [de15bfb](https://github.com/MetaMask/metamask-extension/pull/29356/commits/de15bfb58aa908ad3b7f54092a123b53e963a270)